tensorflow_io/arrow/kernels/arrow_stream_client.h

```diff
@@ -20,10 +20,16 @@ limitations under the License.
 
 namespace tensorflow {
 
+// Supported Address Families
+enum ArrowStreamFamily {
+  AF_INET_SOCKET,
+  AF_UNIX_SOCKET,  // For UNIX systems only
```

Comment:
This is a little different than the `host_type` enum because we are specifically talking about sockets here